backend/tests/setupBvlRoutes.test.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-unused-expressions, prefer-promise-reject-errors */
+const config = require('../config')
+const bvlRoutes = require('../setupBvlRoutes')
+
+const prisonApi = jest.fn()
+const whereaboutsApi = jest.fn()
+const oauthApi = jest.fn()
+const notifyClient = jest.fn()
+const logError = jest.fn()
+const router = { use: jest.fn(), get: jest.fn() }
+
+describe('Test setupBvlRoutes for redirects', () => {
+  beforeEach(() => {
+    jest.resetAllMocks()
+  })
+  it('GET "/videolink" redirect = true, app redirects to http://localhost:3000', async () => {
+    config.app.featureFlags.redirectToBookingVideoLinkEnabled = true
+    bvlRoutes({ prisonApi, whereaboutsApi, oauthApi, notifyClient, logError }, router)
+
+    expect(router.get).toHaveBeenCalledTimes(1)
+    expect(router.use).not.toHaveBeenCalled()
+
+    const [path, handler] = router.get.mock.calls[0]
+    expect(path).toBe('/videolink')
+
+    const res = {
+      redirect: jest.fn(),
+    }
+
+    handler(null, res)
+
+    expect(res.redirect).toHaveBeenCalledWith(config.apis.bookVideoLink.url)
+  })
```

Comment:
This is a bit gnarly but think it's probably acceptable given we'll be removing and simplifying the video link booking code and just leaving a simple redirect that we can but in with the standard redirects